docs/tools/local-testing/era-test-node.md

```diff
@@ -0,0 +1,386 @@
+
+==== 9 events
+EthToken System Contract                   Transfer(address,address,uint256), 0x0000…e23d, 0x0000…8001
+EthToken System Contract                   Transfer(address,address,uint256), 0x0000…e23d, 0x0000…ffe3
+EthToken System Contract                   Transfer(address,address,uint256), 0x0000…ffe3, 0x0000…8908
+0x8a144308792a23aadb118286ac0dec646f638908 Deposit(address,uint256), 0x0000…ffe3
+0x8a144308792a23aadb118286ac0dec646f638908 Transfer(address,address,uint256), 0x0000…ffe3, 0x0000…4b30
+0x880f03ca84e6cf0d0871c9818a2981debaba22b3 Transfer(address,address,uint256), 0x0000…4b30, 0x0000…e23d
+0xc0b7c869ba924c05f64333d9caa21f4424eb4b30 Sync(uint112,uint112)
+0xc0b7c869ba924c05f64333d9caa21f4424eb4b30 Swap(address,uint256,uint256,uint256,uint256,address), 0x0000…ffe3, 0x0000…e23d
+EthToken System Contract                   Transfer(address,address,uint256), 0x0000…8001, 0x0000…e23d
+```
+
+## Sending network calls
+
+You can send network calls against a running `era-test-node`. You can check the testnet LINK balance or mainnet USDT using `curl` or [foundry-zksync](https://github.com/matter-labs/foundry-zksync). 
+
+To get started, launch the local in-memory node:
+
+```bash
+era_test_node fork testnet
+```
+
+Next, use curl to send a network call:
+
+```bash
+curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_call","params":[{"to":"0x40609141Db628BeEE3BfAB8034Fc2D8278D0Cc78", "data":"0x06fdde03"}, "latest"],"id":1}' http://localhost:8011
+```
+
+Here's an example of what you should expect to see:
+
+```bash
+{"jsonrpc":"2.0","result":"0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000018436861696e4c696e6b20546f6b656e2028676f65726c69290000000000000000","id":1}
+```
+
+Or, if you prefer, use [`foundry-zksync`](https://github.com/matter-labs/foundry-zksync). Make sure to install and configure `foundry-zksync` before proceeding (for installation instructions, please see this [link](https://github.com/matter-labs/foundry-zksync/tree/main#foundry-with-zksync-era-v01)):
+
+```bash
+zkcast call 0x40609141Db628BeEE3BfAB8034Fc2D8278D0Cc78 "name()(string)" --rpc-url http://localhost:8011
+```
+
+Here's an example of what you should expect to see:
+
+```bash
+ChainLink Token (goerli)
+```
+
+Retrieve the balance of a particular contract:
+
+```bash
+zkcast call 0x40609141Db628BeEE3BfAB8034Fc2D8278D0Cc78 "balanceOf(address)(uint256)"  0x40609141Db628BeEE3BfAB8034Fc2D8278D0Cc78  --rpc-url http://localhost:8011
+```
+
+Here's an example of what you should expect to see: 
+
+```bash
+28762283719941475444443116625665
+```
+
+### Deploying contracts
+
+Before proceeding, ensure that you've compiled your contracts using `zkforge zk-build`. For instructions on how to do this, please refer to this [link](https://github.com/matter-labs/foundry-zksync#compile-with-zkforge-zk-build).
```

Comment:
Is it possible to deploy via Hardhat? If so, let's mention it first

Reply:
You can deploy via Hardhat but the example makes use of zkforge. I will add Hardhat example as well for parity.